contracts/near/eth-prover/src/lib.rs

```diff
@@ -136,6 +136,71 @@ impl EthProver {
             .into()
     }
 
+    /// WARNING: When the value is not found, `eth_getProof` will return "0x0" at
+    /// the StorageProof `value` field.  In order to verify the proof of non
+    /// existence, you must set `value` to empty vec, *not* the RLP encoding of 0 or null
+    /// (which would be 0x80).
+    #[result_serializer(borsh)]
+    pub fn verify_storage_proof(
+        &self,
+        #[serializer(borsh)] header_data: Vec<u8>,
+        #[serializer(borsh)] account_proof: Vec<Vec<u8>>, // account proof
+        #[serializer(borsh)] contract_address: Vec<u8>,   // eth address
+        #[serializer(borsh)] expected_account_state: Vec<u8>, // encoded account state
+        #[serializer(borsh)] storage_key_hash: Vec<u8>,   // keccak256 of storage key
+        #[serializer(borsh)] storage_proof: Vec<Vec<u8>>, // storage proof
+        #[serializer(borsh)] expected_storage_value: Vec<u8>, // storage value
+        #[serializer(borsh)] min_header_height: Option<u64>,
+        #[serializer(borsh)] max_header_height: Option<u64>,
+        #[serializer(borsh)] skip_bridge_call: bool,
+    ) -> PromiseOrValue<bool> {
+        self.check_not_paused(PAUSE_VERIFY);
+        let header: BlockHeader = rlp::decode(header_data.as_slice()).unwrap();
+
+        if let Some(min_header_height) = min_header_height {
+            if header.number < min_header_height {
+                env::log_str("block height in header data < min header height");
```

Comment:
Let's add the value of header.number and min_header_height to log

Reply:
fixed in 8e864e26e95fb4cd567c6760c84705c590185f4e